spec/git-spec.js

```diff
@@ -424,8 +424,7 @@ describe('git', () => {
         const filePath = path.join(repo.getWorkingDirectory(), 'a.txt')
         fs.writeFileSync(filePath, 'changing a.txt', 'utf8')
         expect(repo.checkoutHead('a.txt')).toBe(true)
-        const lineEnding = process.platform === 'win32' ? '\r\n' : '\n'
-        expect(fs.readFileSync(filePath, 'utf8')).toBe(`first line${lineEnding}`)
+        expect(fs.readFileSync(filePath, 'utf8').replace('\r', '')).toBe(`first line\n`)
```

Comment:
I think if this test started failing, the cause should be investigated, and the test should not be updated unless there is good proof that the test is wrong. Otherwise it seems to me that the package might have a genuine bug on this branch, what with all the dependency changes.

Recording the correct format of line ending is a very important aspect of a git client. So I don't think test should be changed lightly.

Reply:
Removing `\r` normalizes the file across different operating systems. There is nothing wrong with this.

Reply:
Thank you. I had a vague idea maybe AppVeyor had an unusual configuration, so now that you changed that and restored the test this looks right to me.